tools/get-flux-update-pr.sh

```diff
@@ -0,0 +1,43 @@
+latestFluxVersion="$(curl -s --request GET --url "https://api.github.com/repos/fluxcd/flux2/releases?per_page=1" | jq . | jq '.[0] | .tag_name' | jq -r | sed -e 's/v//')"
+
+currentFluxVersion="$(sed -n 's/version="\([0-9\.]*\)"/\1/p' tools/dependencies.toml | head -1)"
+
+echo "Latest flux version $latestFluxVersion"
+echo "Current flux version $currentFluxVersion"
+
+if [[ "$latestFluxVersion" == "$currentFluxVersion" ]]; then
+  echo "No changes needed"
+  exit 0
+fi
+
+newContent=$(sed -e "/version\=\"[0-9\.]*\"/{s//version=\"$latestFluxVersion\"/;:a" -e '$!N;$!ba' -e '}' tools/dependencies.toml)
```

Comment:
I think this has the same issue in that it would fail if we reordered the file. Maybe something like:
`newContent=$(cat /tmp/dependencies.toml | tr '\n' '\r' | sed -e "/\[flux\]\rversion\=\"[0-9\.]*\"/{s//\[flux\]\rversion=\"$latestFluxVersion\"/;:a" -e '$!N;$!ba' -e '}' | tr '\r' '\n')`

Reply:
just changed `/tmp` with `/tools` and worked perfect, thanks 🙂